File: app/src/main/java/com/pkmnapps/activitydo/custominterfaces/TaskActivityInterface.java
package com.pkmnapps.activitydo.custominterfaces;

import com.pkmnapps.activitydo.dataclasses.Widget;

public interface TaskActivityInterface {

    void editWidget(Widget widget);
    void deleteWidget(Widget widget);

    void changeActivtyOfWidget(int type,String uid, int pos);
}

File: app/src/main/java/com/pkmnapps/activitydo/custominterfaces/TaskActivityInterface.kt
package com.pkmnapps.activitydo.custominterfaces

import com.pkmnapps.activitydo.dataclasses.Widget
import org.junit.runner.RunWith

interface TaskActivityInterface {
    open fun editWidget(widget: Widget?)
    open fun deleteWidget(widget: Widget?)
    open fun changeActivtyOfWidget(type: Int, uid: String?, pos: Int)
}